### PR TITLE
use origin-release images for tooling

### PR DESCRIPTION
**Pull request by paulfantom**
Use redhat tooling image

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -56,7 +56,7 @@ integration:
 	go test -v sigs.k8s.io/cluster-api-provider-aws/test/integration
 
 fmt:
-	hack/verify-gofmt.sh
+	hack/go-fmt.sh .
```
Comment (frobware) [inline Makefile]: Is go-fmt different to go-vet w.r.t. passing `./...`. One does, the other doesn't. Curious as to why.

File: hack/go-fmt.sh [hunk 2]
```diff
@@ -9,7 +9,6 @@ else
     --env IS_CONTAINER=TRUE \
     --volume "${PWD}:/go/src/sigs.k8s.io/cluster-api-provider-aws:z" \
     --workdir /go/src/sigs.k8s.io/cluster-api-provider-aws \
-    --entrypoint sh \
-    quay.io/coreos/golang-testing \
+    openshift/origin-release:golang-1.10 \
     ./hack/go-fmt.sh "${@}"
```
Comment (frobware) [inline hack/go-fmt.sh]: Why `${@}` and not just `"$@"`?

Comment (paulfantom) [reply hack/go-fmt.sh]: ShellCheck says that both are valid.

File: Makefile (hunk 1 above)
Comment (paulfantom) [reply Makefile]: No idea, that's how installer runs it.
All those scripts are created by guys from installer team, we just copy-pasted them.

Comment (enxebre) [reply Makefile]: You just need to look at the code of the script https://github.com/openshift/cluster-api-provider-aws/blob/master/hack/go-fmt.sh#L4 https://golang.org/cmd/gofmt/

Comment (enxebre) [reply Makefile]: You just need to look at the code of the script https://github.com/openshift/cluster-api-provider-aws/blob/master/hack/go-fmt.sh#L4 https://golang.org/cmd/gofmt/

**Pull request merged**